ui/pages/remote-mode/overview/remote-mode-permissions.component.tsx

```diff
@@ -54,6 +78,31 @@ export default function RemoteModePermissions({
     account: selectedAccount.address as `0x${string}`,
   });
 
+  useEffect(() => {
+    async function fetchDelegations() {
```

Comment:
We have `remoteModeConfig` object on `useRemoteMode` that returns the swap and daily allowance (the delegation and the metadata). Can we use the hook here to get the delegations and after, the delegation hash?

Reply:
Great suggestion! Switched to grab everything from `remoteModeConfig` (via the `useRemoteMode` hook) in b150814a68765104fa1befaea4ee71c088be0156.